src/icons/AlertWithBackground.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react'
```

Comment:
instead of adding a new icon, can we use the existing Alert icon and draw the pink background with css? i feel like our icons folder is becoming massive and difficult to navigate, also it seems reasonable to assume that if the Alert icon needs to be updated then we'd also expect that change to affect the homefeed icon so i was thinking they should be the same component?